editor/input_map_editor.cpp

```diff
@@ -36,44 +36,44 @@
 #include "editor/editor_scale.h"
 
 static const char *_button_descriptions[JOY_SDL_BUTTONS] = {
-	"Face Bottom, DualShock Cross, Xbox A, Nintendo B",
-	"Face Right, DualShock Circle, Xbox B, Nintendo A",
-	"Face Left, DualShock Square, Xbox X, Nintendo Y",
-	"Face Top, DualShock Triangle, Xbox Y, Nintendo X",
-	"DualShock Select, Xbox Back, Nintendo -",
-	"Home, DualShock PS, Guide",
-	"Start, Nintendo +",
-	"Left Stick, DualShock L3, Xbox L/LS",
-	"Right Stick, DualShock R3, Xbox R/RS",
-	"Left Shoulder, DualShock L1, Xbox LB",
-	"Right Shoulder, DualShock R1, Xbox RB",
-	"D-Pad Up",
-	"D-Pad Down",
-	"D-Pad Left",
-	"D-Pad Right"
```

Comment:
I think this one could benefit from adding context to TTRC, e.g.
TTRC("Face Bottom, DualShock Cross, Xbox A, Nintendo B", "Button_Input_Description");